scss/_alert.scss

```diff
@@ -27,15 +27,14 @@
 // Expand the right padding and account for the close button's positioning.
 
 .alert-dismissible {
-  padding-right: $close-font-size + $alert-padding-x * 2;
+  padding-right: $alert-padding-x * 3.5;
```

Comment:
`3.5` and `1.25` above look  a bit random here. Does it match anything? Shouldn't they be handled as variables?